shared/src/akri/configuration.rs

```diff
@@ -134,53 +122,48 @@ pub async fn get_configurations(
 ///
 /// ```no_run
 /// use akri_shared::akri::configuration;
-/// use kube::client::APIClient;
+/// use kube::client::Client;
 /// use kube::config;
 ///
 /// # #[tokio::main]
 /// # async fn main() {
-/// let api_client = APIClient::new(config::incluster_config().unwrap());
-/// let dcc = configuration::find_configuration(
```

Comment:
lol ... dcc ... devcapconfig :)

Reply:
What a gem. I excavated it 67595e71132f :)